tools/paquet/.gitignore

```diff
@@ -0,0 +1,6 @@
+/.bundle/
```

Comment:
what happens when the first character is `/`? I assume we're not hoping to see a `.bundle` at the root of the filesystem?